kombu/transport/redis_cluster.py

```diff
@@ -342,8 +349,7 @@ def _brpop_read(self, **options):
             return True
 
     def _poll_error(self, cmd, conn, **options):
-        if cmd == 'BRPOP':
-            conn.client.parse_response(conn.client.connection, cmd, **options)
+        self.connection.cycle._unregister((self, self.client, conn, 'BRPOP'))
```

Comment:
i believe we should not remove `parse_response` part since we should know exception is raised